server/src/main/java/io/deephaven/server/table/ops/ComboAggregateGrpcImpl.java

```diff
@@ -109,8 +111,9 @@ private boolean isSimpleAggregation(ComboAggregateRequest request) {
     }
 
     @Override
-    public Table create(final ComboAggregateRequest request,
+    public Table create(final AuthContext authContext, final ComboAggregateRequest request,
             final List<SessionState.ExportObject<Table>> sourceTables) {
```

Comment:
Presumably, if I can reference something that is in sourceTable's I've already gotten an export for it, and any mutations (row/column filtering) that authorization would apply have been applied?